starter/source/tools/sensor/hm_read_sensors.F

```diff
@@ -191,87 +182,83 @@ SUBROUTINE HM_READ_SENSORS(
             CALL READ_SENSOR_ACC(SENSOR_PTR ,SENS_ID  ,TITR     , 
-     .           LSENS_PTR ,RSENS_PTR  ,LACCELM  ,UNITAB   ,LSUBMODEL)  
+     .           LACCELM  ,UNITAB   ,LSUBMODEL)  
 c
           CASE ('DIST' ,'TYPE2')
             CALL READ_SENSOR_DISP(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,ITABM1   ,UNITAB   ,LSUBMODEL)   
+     .           ITABM1   ,UNITAB   ,LSUBMODEL)   
 c
           CASE ('SENS' ,'TYPE3')
-            CALL READ_SENSOR_SENS(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+            CALL READ_SENSOR_SENS(SENSOR_PTR ,SENS_ID  ,UNITAB   ,LSUBMODEL)
 c
           CASE ('AND' ,'TYPE4')
-            CALL READ_SENSOR_AND(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+            CALL READ_SENSOR_AND(SENSOR_PTR ,SENS_ID  ,UNITAB   ,LSUBMODEL)
 c
           CASE ('OR' ,'TYPE5')
-            CALL READ_SENSOR_OR(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+            CALL READ_SENSOR_OR(SENSOR_PTR ,SENS_ID  ,UNITAB   ,LSUBMODEL)
 c
           CASE ('INTER' ,'TYPE6')
             CALL READ_SENSOR_CONTACT(SENSOR_PTR ,SENS_ID  ,TITR     ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+     .           UNITAB   ,LSUBMODEL)
 c
           CASE ('RWALL' ,'TYPE7')
             CALL READ_SENSOR_RWALL(SENSOR_PTR ,SENS_ID  ,TITR     ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+     .           UNITAB   ,LSUBMODEL)
 c
           CASE ('NOT' ,'TYPE8')
-            CALL READ_SENSOR_NOT(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+            CALL READ_SENSOR_NOT(SENSOR_PTR ,SENS_ID  ,UNITAB   ,LSUBMODEL)
 c
           CASE ('VEL' ,'TYPE9')
             CALL READ_SENSOR_VEL(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,ITABM1   ,UNITAB   ,LSUBMODEL)   
+     .           ITABM1   ,UNITAB   ,LSUBMODEL)   
 c
           CASE ('GAUGE' ,'TYPE10')            
             CALL READ_SENSOR_GAUGE(SENSOR_PTR ,SENS_ID  ,TITR     ,
-     .           LGAUGE   ,LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+     .           LGAUGE   ,UNITAB   ,LSUBMODEL)
 c
           CASE ('RBODY' ,'TYPE11')
             CALL READ_SENSOR_RBODY(SENSOR_PTR ,SENS_ID  ,TITR     ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+     .           UNITAB   ,LSUBMODEL)
 c
           CASE ('SECT' ,'TYPE12')
             CALL READ_SENSOR_SECT(SENSOR_PTR ,SENS_ID  ,TITR     ,
-     .           LSENS_PTR ,RSENS_PTR  ,UNITAB   ,LSUBMODEL)
+     .           UNITAB   ,LSUBMODEL)
 c
           CASE ('WORK' ,'TYPE13')
             CALL READ_SENSOR_WORK(SENSOR_PTR ,SENS_ID  ,
-     .           LSENS_PTR ,RSENS_PTR  ,ITABM1   ,UNITAB   ,LSUBMODEL)   
+     .           ITABM1   ,UNITAB   ,LSUBMODEL)   
 c
           CASE ('ENERGY' ,'TYPE14')
             CALL READ_SENSOR_ENERGY(SENSOR_PTR ,SENS_ID  ,TITR     ,
```

Comment:
Here call to the routine is modified , but not routine -> this may lead to forcheck issue .